openlibrary/plugins/upstream/account.py

```diff
@@ -901,6 +901,28 @@ def POST(self):
                 books_wo_isbns[_book['Book Id']] = _book     
         return render['account/import'](books, books_wo_isbns)
 
+class export_books(delegate.page):
+    path = "/account/export"
+
+    @require_login
+    def GET(self):
+        user = accounts.get_current_user()
+        username = user.key.split('/')[-1]
+        books = Bookshelves.get_users_logged_books(username, limit=10000)
```

Comment:
It would be nice if this `limit` defaulted to `None`. Not a blocker; 10k seems like an arbitrary number